tests/http_spec.rs

```diff
@@ -255,7 +255,10 @@ async fn assert_downstream(spec: HttpSpec) {
 
       // Assert Headers
       for (key, value) in assertion.response.0.headers.iter() {
-        assert_eq!(actual_headers.get(key), Some(value));
+        match actual_headers.get(key) {
+          None => panic!("Expected header {} to be present. But not found", key),
```

Comment:
```suggestion
          None => panic!("Expected header {} to be present", key),
```
nitpik: just to making it concise.